### PR TITLE
Regenerate using latest swagger.

### PR DESCRIPTION
**Pull request by JoshLove-msft**

### CODE REVIEW

File: sdk/schemaregistry/Azure.Data.SchemaRegistry/src/autorest.md [hunk 1]
```diff
@@ -3,7 +3,6 @@
 Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
-# From github.com/Azure/azure-rest-api-specs/tree/arerlend.sr.init
-# require: https://github.com/Azure/azure-rest-api-specs/blob/1e23d91e875e4464e57667639e06408cef99868d/specification/schemaregistry/data-plane/readme.md
-input-file: schemaregistry.json
+title: SchemaRegistryClient
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/72fbc9e93123bbfa8cb44042fd649e80870ce5b2/specification/schemaregistry/data-plane/Microsoft.EventHub/stable/2021-10/schemaregistry.json
```
Comment (pakrym) [inline sdk/schemaregistry/Azure.Data.SchemaRegistry/src/autorest.md:7]: I think there was an effort to convert all of these to README.md references.

File: sdk/schemaregistry/Azure.Data.SchemaRegistry/tests/SchemaRegistryClientLiveTests.cs [hunk 2]
```diff
@@ -11,7 +11,7 @@ namespace Azure.Data.SchemaRegistry.Tests
 {
     public class SchemaRegistryClientLiveTests : RecordedTestBase<SchemaRegistryClientTestEnvironment>
     {
-        public SchemaRegistryClientLiveTests(bool isAsync) : base(isAsync)
+        public SchemaRegistryClientLiveTests(bool isAsync) : base(isAsync, RecordedTestMode.Record)
```
Comment (pakrym) [inline sdk/schemaregistry/Azure.Data.SchemaRegistry/tests/SchemaRegistryClientLiveTests.cs:14]: ...

**Review by pakrym — APPROVED** [review]

File: sdk/schemaregistry/Azure.Data.SchemaRegistry/src/autorest.md (hunk 1 above)
Comment (JoshLove-msft) [reply sdk/schemaregistry/Azure.Data.SchemaRegistry/src/autorest.md:7]: Will do once the final swagger update is merged.

File: sdk/schemaregistry/Azure.Data.SchemaRegistry/src/SchemaFormat.cs [hunk 3]
```diff
@@ -1,15 +1,44 @@
...
+        /// <exception cref="ArgumentNullException"> <paramref name="value"/> is null. </exception>
+        public SchemaFormat(string value)
+        {
+            _value = value ?? throw new ArgumentNullException(nameof(value));
+        }
+
+        private const string AvroValue = "avro";
+
+        /// <summary> Avro Serialization schema type. </summary>
+        public static SchemaFormat Avro { get; } = new SchemaFormat(AvroValue);
```
Comment (swathipil) [inline sdk/schemaregistry/Azure.Data.SchemaRegistry/src/SchemaFormat.cs:24]: just wanted to confirm that we accept case insensitive Avro in registerSchema/getSchemaProperties and set SchemaProperties.format = "Avro" when we return SchemaProperties?

Comment (JoshLove-msft) [reply sdk/schemaregistry/Azure.Data.SchemaRegistry/src/SchemaFormat.cs:24]: No, it just always uses "avro". What do you mean when you say the service returns "Avro"? I thought it was just part of the content-type as serialization=avro.

Comment (JoshLove-msft) [reply sdk/schemaregistry/Azure.Data.SchemaRegistry/src/SchemaFormat.cs:24]: It shouldn't make a difference here as we use case-insensitive comparison for the SchemaFormat type.

Comment (JoshLove-msft) [reply sdk/schemaregistry/Azure.Data.SchemaRegistry/src/SchemaFormat.cs:24]: Regenerated with "Avro"

**Pull request merged**